lib/storage/file.js

```diff
@@ -1019,12 +1019,18 @@ File.prototype.getSignedPolicy = function(options, callback) {
  *     link will expire.
  * @param {string=} options.extensionHeaders - If these headers are used, the
  *     server will check to make sure that the client provides matching values.
+ * @param {string=} options.responseDisposition - If you provide this value, 
```

Comment:
Can you also explain why the user may want to set these two parameters in the first place (i.e. so they don't need to find additional documentation elsewhere).